lua/entities/gmod_wire_expression2/core/custom/cfc_array.lua

```diff
@@ -90,3 +90,20 @@ e2function number array:indexOf(vector4 vec4)
     return indexOfPackedValue( this, vec4 )
 end
 
+e2function number array:indexOf(vector4 vec4)
+    return indexOfPackedValue( this, vec4 )
+end
+
+registerFunction( "first", "r:", id, function(self,args)
+  local op1 = args[2]
```

Comment:
could you use more descriptive variable names

Reply:
i did just use the shift function as a template, cut me some slack, anyway will do